sample-app/src/components/ResultsCount.tsx

```diff
@@ -1,25 +1,69 @@
 import { useAnswersState } from '@yext/answers-headless-react';
-export interface ResultsCountConfig {
-  resultsCount?: number,
-  resultsLength?: number,
-  offset?: number
+import classNames from 'classnames';
+import { CompositionMethod, useComposedCssClasses } from '../hooks/useComposedCssClasses';
+
+interface ResultsCountCssClasses {
+  container?: string,
+  text?: string,
+  number?: string
 }
 
-export function ResultsCountDisplay(props: ResultsCountConfig): JSX.Element {
-  const resultsCount = props.resultsCount || 0;
-  const resultsLength = props.resultsLength || 0;
-  const offset = props.offset || 0;
-  const results = ((resultsLength === 0) ? "" : `${offset + 1}-${offset + resultsLength} of ${resultsCount}`)
+const builtInCssClasses: ResultsCountCssClasses = {
+  container: 'pb-4',
+  text: 'text-sm text-gray-700',
+  number: 'font-medium'
+}
+
+interface Props {
+  customCssClasses?: ResultsCountCssClasses,
+  cssCompositionMethod?: CompositionMethod
+}
 
-  return (
-    <div> {results} </div>
-  )
+export interface ResultsCountConfig {
+  resultsCount?: number,
+  resultsLength?: number,
+  offset?: number,
+  cssClasses?: ResultsCountCssClasses
 }
 
 
-export default function ResultsCount() {
+export default function ResultsCount({ customCssClasses, cssCompositionMethod }: Props) {
   const resultsCount = useAnswersState(state => state.vertical?.resultsCount) || 0;
   const resultsLength = useAnswersState(state => state.vertical?.results?.length) || 0;
   const offset = useAnswersState(state => state.vertical?.offset) || 0;
-  return <ResultsCountDisplay resultsCount={resultsCount} resultsLength={resultsLength} offset={offset}/>;
+  const cssClasses = useComposedCssClasses(builtInCssClasses, customCssClasses, cssCompositionMethod);
+  return <ResultsCountDisplay resultsCount={resultsCount} resultsLength={resultsLength} offset={offset} cssClasses={cssClasses}/>;
 }
+
+export function ResultsCountDisplay({
```

Comment:
there may be other classes that use ResultsCountDisplay directly (like SectionHeader), what do you think about letting ResultsCount pass in the customCssClass and  ResultsCountDisplay can be the one to call `useComposedCssClasses` to include builtInCssClasses

Reply:
Sure, I updated this to compose in the display component.